utils/auth/Cargo.toml

```diff
@@ -13,6 +13,7 @@ rand         = "0.8"
 serde        = { version = "1.0", default-features = false, features = ["derive"] }
 thiserror    = "1.0"
 # Internal
+anyhow                = "=1.0.59"
```

Comment:
Is this still required?

Reply:
Yes. `anyhow` is the reason of our. problems

Reply:
Add the reason why `anyhow` is required as a comment in the `Cargo.toml` file. Right now the dependency looks unused and could easily be deleted.